magic_build_and_package.sh

```diff
@@ -7,147 +11,130 @@
 set -e
 set -x
 
-if [ -z "$LANG" ]; then
-  LANG='en-US'
-fi
-
+BASEDIR=$(dirname $0)
 VERBOSE=false #TODO
 
 while [[ $# > 1 ]]
 do
-    key="$1"
-
-    case $key in
-        -lang|--language)
-        LANG="$2"
-        shift # past argument
-        ;;
-        -v|--verbose)
-        VERBOSE=true
-        ;;
-        *)
-        echo "unknown option $key"
-        exit
-        ;;
-    esac
-    shift # past argument or value
+  key="$1"
+
+  case $key in
+    -lang|--language)
+    LANG="$2"
+    shift # Consume additional argument
+    ;;
+
+    -v|--verbose)
+    VERBOSE=true
+    ;;
+
+    *)
+    echo "Unknown option $key"
+    exit 1
+    ;;
+  esac
+  shift # Consume current argument
 done
 
 if [[ "$OSTYPE" == "linux-gnu" ]]; then
-    export IS_LINUX=true
-    echo 'Linux OS detected'
+  IS_LINUX=true
+  echo 'Linux OS detected'
 elif [[ "$OSTYPE" == "darwin"* ]]; then
-    export IS_MAC_OS=true
-    echo 'MAC OS detected'
-elif [[ "$OSTYPE" == "cygwin" ]]; then
-    export IS_WIN=true
-    echo 'WINDOWS OS detected'
-elif [[ "$OSTYPE" == "msys" ]]; then
-    export IS_WIN=true
-    echo 'WINDOWS OS detected'
+  IS_MAC_OS=true
+  echo 'Mac OS detected'
+elif [[ "$OSTYPE" == "cygwin" || "$OSTYPE" == "msys" ]]; then
+  IS_WIN=true
+  echo 'Windows OS detected'
 else
-    echo 'Unknow OS -`$OSTYPE`'
+  echo 'Unknow OS -`$OSTYPE`'
 fi
 
-echo "Starting build on with language $LANG and VERBOSE=$VERBOSE"
-
 cd mozilla-release
 
-export CQZ_VERSION=$(awk -F "=" '/version/ {print $2}' ../repack/distribution/distribution.ini | head -n1)
-export CQZ_UI_LOCALE=`echo $LANG`
-export MOZ_OBJDIR=obj-firefox
-export MOZCONFIG=`pwd`/browser/config/mozconfig
-export MOZ_AUTOMATION_UPLOAD=1
+export MOZCONFIG=browser/config/cliqz-release.mozconfig
```

Comment:
Well, it works now :)
The script sets directory on line 52. Not sure that it won't break if run from a different directory.
Anyway, I tried it, and absolute path seems to break multiarch build for some reason. I don't feel like investigating it again now.